depclean-gradle-plugin/src/main/java/se/kth/depclean/analysis/GradleProjectDependencyAnalysis.java

```diff
@@ -0,0 +1,144 @@
+package se.kth.depclean.analysis;
+
+import java.util.Collections;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import org.gradle.api.artifacts.ResolvedArtifact;
+import org.gradle.api.component.Artifact;
+
+public class GradleProjectDependencyAnalysis {
+
+  /**
+   * Store all the used declared artifacts (ie. used direct dependencies).
+   */
+  private final Set<ResolvedArtifact> usedDeclaredArtifacts;
+
+  /**
+   * Store all the used undeclared artifacts (ie. used transitive dependencies).
+   */
+  private final Set<ResolvedArtifact> usedUndeclaredArtifacts;
+
+  /**
+   * Store all the unused declared artifacts (ie. unused transitive dependencies).
+   */
+  private final Set<ResolvedArtifact> unusedDeclaredArtifacts;
+
+  /**
+   * Ctor.
```

Comment:
to remove

Reply:
Are you talking about that comment here?